targets/psa_switch/primitives.cpp

```diff
@@ -398,6 +398,22 @@ class truncate_ : public ActionPrimitive<const Data &> {
 
 REGISTER_PRIMITIVE_W_NAME("truncate", truncate_);
 
+class assert_ : public ActionPrimitive<const Data &> {
```

Comment:
I don't understand why these are not added to the core primitives: https://github.com/p4lang/behavioral-model/blob/master/src/bm_sim/core/primitives.cpp

Reply:
I have read that l2_switch and simple_router don't work with P416 since last year and that is why I put it separated previously. Now, I move them in core/primitives.cpp